app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tab/Ajax/Serializer.php

```diff
@@ -5,9 +5,37 @@
  */
 namespace Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Ajax;
 
+use Magento\Framework\View\Element\Template;
+
+/**
+ * Class Serializer
+ * @package Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Ajax
+ * @deprecated
+ */
 class Serializer extends \Magento\Framework\View\Element\Template
 {
     /**
+     * @var \Magento\Framework\Serialize\SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * @param Template\Context $context
+     * @param \Magento\Framework\Serialize\SerializerInterface|null $serializer
+     * @param array $data
+     * @throws \RuntimeException
+     */
+    public function __construct(
+        Template\Context $context,
+        \Magento\Framework\Serialize\SerializerInterface $serializer = null,
+        array $data = []
+    ) {
+        parent::__construct($context, $data);
+        $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
* Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`.
* Since you are creating a new constructor, but not modify the existing one – you may use regular dependency injection technique.